### ISSUE TITLE
CalDav does not take free/busy status of event into consideration

### ISSUE THREAD

**Issue by lcx** [calendso/calendso#1101]
### Issue Summary
With Zimbra (and probably other systems) events can be declared as free or busy, this is however not taken into consideration by calendso. 

### Steps to Reproduce

1. create an event and declare it as free
2. check calendso, it will show up as busy

an example entry is this: 

```
BEGIN:VCALENDAR
PRODID:Zimbra-Calendar-Provider
VERSION:2.0
METHOD:PUBLISH
BEGIN:VEVENT
UID:82CB91B8-ADF6-46F4-82AA-D78A90AA44F1
SUMMARY:Some Non Busy Event
X-APPLE-TRAVEL-ADVISORY-BEHAVIOR:AUTOMATIC
DTSTART;VALUE=DATE:20211027
DTEND;VALUE=DATE:20211108
STATUS:CONFIRMED
CLASS:PUBLIC
X-MICROSOFT-CDO-ALLDAYEVENT:TRUE
X-MICROSOFT-CDO-INTENDEDSTATUS:FREE
TRANSP:TRANSPARENT
LAST-MODIFIED:20211101T163709Z
DTSTAMP:20211101T163709Z
SEQUENCE:0
END:VEVENT
END:VCALENDAR
```

this seems to be done via the `X-MICROSOFT-CDO-INTENDEDSTATUS:FREE` curios if office365 uses the same settings.
This would be the last thing I need to be able to finally switch to Calendso.

**Comment by jdaviescoates** [comment]
I came here to report the exact same thing. 

I've connected my Nextcloud CalDav server in Cal.com but on my public profile page it's still possible to book the times where I'm actually already busy, as per my Nextcloud calendar.

**Comment by jdaviescoates** [comment]
It looks like a fix (that'd I'd guess will likely fix this too) is in the works #624

**Comment by ConnorGargano** [comment]
Hi @jdaviescoates, looks like these are in relation with each other and I can see you have noted this issue in the previous one so I will go ahead and close.
-Thanks, Connor
